<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryEdit</class>
 <widget class="QWidget" name="CategoryEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>67</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>67</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="w_remove">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="w_add">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="w_title"/>
   </item>
   <item>
    <widget class="QComboBox" name="w_values">
     <item>
      <property name="text">
       <string>UniqueText</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MultiText</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DateTime</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="w_edit">
     <property name="text">
      <string>Edit value</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
